ext/flash/lib.rs

```diff
@@ -205,8 +205,13 @@ async fn op_flash_write_resource(
   server_id: u32,
   token: u32,
   resource_id: deno_core::ResourceId,
+  auto_close: bool,
 ) -> Result<(), AnyError> {
-  let resource = op_state.borrow_mut().resource_table.take_any(resource_id)?;
+  let resource = if auto_close {
```

Comment:
There was and still is two `op_state.borrow_mut()` calls here one after the other. Would it be possible to combine them?